website/docs/r/event_orchestration_router.html.markdown

```diff
@@ -12,15 +12,31 @@ An Orchestration Router allows users to create a set of Event Rules. The Router
 
 ## Example of configuring Router rules for an Orchestration
 
-In this example the user has defined the Router with two rules, each routing to a different service.
-
-This example assumes services used in the `route_to` configuration already exists. So it does not show creation of service resource.
```

Comment:
**Annotation:** I decided it was easier to include a pair of [service "data source" configurations](https://registry.terraform.io/providers/PagerDuty/pagerduty/latest/docs/data-sources/service) in the example, rather having these disclaimer sentences.